<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BonusMap</class>
 <widget class="QDialog" name="BonusMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Бонусы</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Место</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbRank">
       <property name="suffix">
        <string> место</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbCup">
       <item>
        <property name="text">
         <string>Чемпион</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Финал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Полуфинал</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Четвертьфинал</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Премиальные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sbBonus">
       <property name="suffix">
        <string> pts</string>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Место</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Призовые</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить в таблицу</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BonusMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BonusMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
